OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -296,4 +274,31 @@
             <li>Remove the Tip Binding in the Pen Settings panel by clicking the 3 dots, then pressing clear.</li>
             <li>Save your settings and try drawing in an applicantion that supports pressure.</li>
         </ol>
+    <p>
+        You will also need to use artist mode mouse bindings in the advanced binding editor (press '...' next to the binding)
+        instead of regular mouse buttons. Regular mouse buttons will be ignored.<br/><br/>
+
+        Using artist mode will result in some minor smoothing, this is usually undiserable for users as OTD already gives you
```

Comment:
```suggestion
        Using artist mode will result in some minor smoothing because of libinput, this is usually undesirable for users as OTD already gives you
```